linera-execution/solidity/LineraTypes.sol

```diff
@@ -44,292 +43,313 @@ library LineraTypes {
                     power *= 128;
                 }
                 result += power * uint8(input[pos + idx]);
-                uint256 new_pos = pos + idx + 1;
-                return (new_pos, result);
+                return (pos + idx + 1, result);
             }
             idx += 1;
         }
         require(false, "This line is unreachable");
         return (0,0);
     }
 
-    struct TimeoutConfig {
-        opt_TimeDelta fast_round_duration;
-        TimeDelta base_timeout;
-        TimeDelta timeout_increment;
-        TimeDelta fallback_duration;
+    struct AccountOwner {
+        uint8 choice;
```

Comment:
Maybe `tag`, `discriminant`, `account_type` or `owner_kind`?

And should this be an enum instead?

Reply:
That makes sense, yes.
I will have to think about the right way to do it in `serde-reflection`, but yes, it can be done.